index.html

```diff
@@ -164,6 +164,22 @@ <h3>H3 Component Library</h3>
               <a class="more" href="#"><i class="fas fa-ellipsis-v"></i></a>
             </div>
           </div>
+        <!-- Rounded Image -->
+        <img
+            class="img img-rounded img-10"
+            src="https://material-components.github.io/material-components-web-catalog/static/media/photos/3x2/2.jpg"
+            srcset="
```

Comment:
you can add a loading="lazy" attribute to `img` tag. That loads image only when it comes to viewport. Just a suggestion. Otherwise all is good.

Reply:
Noted. Thanks for the suggestion.